content/en/docs/tutorials/dynamic-resource-allocation/comapring-dra-device-plugin.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Comparing Dynamic Resource Allocation to Device Plugins
+content_type: tutorial
+weight: 10
+---
+
+<!-- overview -->
+
+Both Dynamic Resource Allocation (DRA) and device plugins enable Kubernetes
+workloads to leverage specialized hardware from various vendors. This tutorial
+will show how to configure the same GPU-enabled workload with DRA and device
+plugins to illustrate the differences between the two sets of APIs.
+
+
+## {{% heading "objectives" %}}
+
+* Learn when to prefer using device plugins or DRA when configuring containers'
+  requests for devices.
+
+
+## {{% heading "prerequisites" %}}
+
+* An NVIDIA GPU-enabled cluster with GPU Operator installed
```

Comment:
Something vendor-specific like this might not fit well in these docs, but NVIDIA GPUs are probably the most common DRA use case right now and NVIDIA's device plugin and DRA driver make for the most apples-to-apples comparison of these APIs at the moment I think.

Reply:
Can we use [tabs](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs) to let people take part even with different vendors?

Reply:
I think that would work well if we can map the same use cases 1:1 across vendors, like if AMD eventually exposes a similar GPU and "security/network isolation" device like NVIDIA's IMEX channels. Different use cases might be better expressed as separate sections though, but we can definitely play around with how those look when we can produce more examples here.